setup.cfg

```diff
@@ -83,7 +84,7 @@ version = 0.20.1
 banzai = banzai.main:main
 banzai_reduce_individual_frame = banzai.main:reduce_single_frame
 banzai_reduce_directory = banzai.main:reduce_directory
-banzai_stack_calibrations = banzai.main:stack_calibrations
```

Comment:
Do we maybe want to retain a method for stacking calibrations locally, i.e. an entrypoint that just calls `process_master_maker(...)` without the `.send` (I think this would work...)

Reply:
You're completely right--I've added a wrapper function with an entrypoint that calls process_master_maker.